java/src/test/java/org/rocksdb/EnvOptionsTest.java

```diff
@@ -0,0 +1,135 @@
+package org.rocksdb;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EnvOptionsTest {
+
+    @ClassRule
+    public static final RocksMemoryResource rocksMemoryResource =
+            new RocksMemoryResource();
+
+    public static final Random rand = PlatformRandomHelper.
+            getPlatformSpecificRandomFactory();
+
```

Comment:
Please remove the `set` prefix from your test names so that they are consistent with the existing tests such as `OptionsTest` and `DBOptionsTest` etc.